hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/EventExpansion.java

```diff
@@ -59,7 +59,7 @@ public void expandAllSigs(final Event event, final ServicesState sourceState) {
                         // synchronous signature expansion
                         prefetchProcessor.submit(accessor);
                         sigReqsManager.expandSigs(sourceState, accessor);
-                        engine.verifyAsync(txn.getSignatures());
+                        engine.verifyAsync(accessor.getCryptoSigs());
```

Comment:
Why are we not using `txn.getSignatures()` ? Is it just because both are same or `txn.getSignatures()` doesn't work here ?

Reply:
The initial motivation was to reduce exposure to the `Transaction` interface in unit tests.

But it's also worth noting that since `SwirldTransaction.addAll()` only accepts an **array** of `TransactionSignature`'s, we were creating a gratuitous array [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/Expansion.java#L96) for every transaction! 

This also avoids creating that array.